pkg/ebpf/bytecode/runtime/conntrack.go

```diff
@@ -4,4 +4,4 @@
 
 package runtime
 
-var Conntrack = NewRuntimeAsset("conntrack.c", "a72a9cf2b321f8825b1733b72e264a5df37c1aa280f0ff65c3900a5df84c993c")
```

Comment:
We usually don't commit those changes when there is no edit to a related C file. They generally change when you run `inv -e system-probe.build` and the hash may depends on the installed clang version (which is different depending on teams/users it seems).

Reply:
resolved on last commit